packages/migrations/src/createStakerNetworkAndConnectStakerPkgs.ts

```diff
@@ -0,0 +1,42 @@
+import { docker } from "@dappnode/dockerapi";
+import { params } from "@dappnode/params";
+import { logs } from "@dappnode/logger";
+import { ensureStakerPkgsNetworkConfig } from "@dappnode/stakers";
+import { networks } from "@dappnode/types";
+
+/**
+ * Creates the staker network and connects the staker packages to it
+ */
+export async function createStakerNetworkAndConnectStakerPkgs(): Promise<void> {
```

Comment:
In general, functions should have a single purpose. Including "and" in a function name is indicating something is wrong. 

Maybe they can be called independently or creating the staker network could be a pre requisite